README.md

```diff
@@ -1,6 +1,6 @@
-<img src="http://parse.com/assets/svgs/parse-infinity.svg" alt="Parse logo" height="70"/>
+![Parse Server logo](.github/parse-server-logo.png?raw=true)
```

Comment:
can you edit the image to remove the padding on top/bottom?.. 

Then I think the headline below can go away?